### PR TITLE
Update to ESLint 1.0

### PR DESCRIPTION
**Pull request by pdehaan**
Current output:

```
➜  review.ninja git:(master) ✗ npm run lint

> ReviewNinja@1.2.0 lint /Users/pdehaan/dev/tmp/review.ninja
> gulp lint

[09:55:24] Using gulpfile ~/dev/tmp/review.ninja/gulpfile.js
[09:55:24] Starting 'lint'...
[09:55:24] Starting 'eslint'...
[09:55:27] Finished 'eslint' after 3.46 s
[09:55:27] Finished 'lint' after 3.46 s

➜  review.ninja git:(master) ✗ eslint .
➜  review.ninja git:(master) ✗ eslint --version
v1.0.0
```

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/reviewninja/review.ninja/pull/952" target="_blank">ReviewNinja

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
+extends:
+    - eslint:recommended
```
Comment (pdehaan) [inline .eslintrc]: By default ESLint@1 doesn't enable any rules, so this just brings in their "recommended" set of default rules. But the nice thing is that you can extend multiple rule sets, so it's pretty easy to extend the recommended rules and use AirBnB's Style Guide as well.

File: .eslintrc [hunk 2]
```diff
@@ -21,20 +24,21 @@ rules:
...
+    no-shadow: 0
     no-trailing-spaces: 1
     no-undef: 0
     no-underscore-dangle: 0
     no-unused-expressions: 0
     no-unused-vars: 0
     no-use-before-define: [1, nofunc]
-    no-empty: 0
     quotes: [1, single]
+    semi-spacing: 1
```
Comment (pdehaan) [inline .eslintrc]: This was just the renamed `no-space-before-semi: 1` rule.

File: .eslintrc [hunk 3]
```diff
@@ -21,20 +24,21 @@ rules:
...
     semi: 2
     space-infix-ops: 1
-    strict: 1
+    strict: [1, global]
```
Comment (pdehaan) [inline .eslintrc]: I added the "global" but to silence the 300 _"Use the function form of 'use strict'"_ warnings.

**Comment by dfarr** [comment]
Thank you @pdehaan you are a QA master :+1:

**Pull request merged**